hubmap/frontend/cypress/integration/hubmap_spec.js

```diff
@@ -15,7 +32,6 @@ describe('HuBMAP', () => {
     // Header:
     cy.contains('Browse');
     cy.contains('Help');
-    cy.contains('Login');
```

Comment:
Replace this with an assertion about what is present instead. (And then click on it to expand the menu, and make assertions on the menu contents.)

Reply:
handled above.